torch/_inductor/output_code.py

```diff
@@ -334,6 +340,64 @@ def get_constants(
         }
         return constants
 
+    def prepare_for_serialization(self) -> None:
+        # We can't really serialize callables that may be C++/Triton/etc.,
+        # so we serialize their PyCodeCache disk cache location instead.
+        # TODO: This could be better if we're ever able to serialize compiled
+        # models to disk.
+        self.current_callable = None
+
+    def after_deserialization(self, gm: Optional[torch.fx.GraphModule]) -> str:
```

Comment:
@ezyang , this is basically just pulling out the code you wanted to refactor into post_compile, right?

Reply:
Also qq (maybe for @aorenste) do we do this serialization even when the outputcode is generated via AOTI (not CompiledFxGraph)? I.e. does it belong in OutputCode, or does it belong in CompiledFxGraph?

Reply:
I'm not sure why we want this separate from post_compile (there's probably a good reason, I'm just not seeing it). I am not sure why this isn't a shared method in the top level protocol.

Reply:
There may not be a good reason. I did all this code before post_compile/OutputCode existed - so it may make sense to merge it all together better.